matminer/featurizers/structure/__init__.py

```diff
@@ -0,0 +1,49 @@
+from matminer.featurizers.structure.composition import (
+    StructureComposition
+)
+from matminer.featurizers.structure.sites import (
+    SiteStatsFingerprint
+)
+from matminer.featurizers.structure.voronoi import (
+    StructuralHeterogeneity,
+    MaximumPackingEfficiency,
+    ChemicalOrdering
+)
+from matminer.featurizers.structure.distribution import (
+    RadialDistributionFunction,
+    PartialRadialDistributionFunction,
+    ElectronicRadialDistributionFunction,
+    XRDPowderPattern
+)
+from matminer.featurizers.structure.matrix import (
+    CoulombMatrix,
+    SineCoulombMatrix,
+    OrbitalFieldMatrix,
+)
+from matminer.featurizers.structure.bonding import (
+    BagofBonds,
+    BondFractions,
+)
+from matminer.featurizers.structure.symmetry import (
+    GlobalSymmetryFeatures,
+    Dimensionality
+)
+from matminer.featurizers.structure.misc import (
+    GlobalInstabilityIndex,
+    StructuralComplexity,
+    MinimumRelativeDistances,
+    DensityFeatures,
+    EwaldEnergy,
+)
+from matminer.featurizers.structure.jarvis import (
+    JarvisCFID
+)
+from matminer.featurizers.structure.cgcnn import (
+    CGCNNFeaturizer
+)
+
+__authors__ = ("Anubhav Jain <ajain@lbl.gov>, "
+                "Saurabh Bajaj <sbajaj@lbl.gov>, "
+                "Nils E.R. Zimmerman <nils.e.r.zimmermann@gmail.com>, "
+                "Alex Dunn <ardunn@lbl.gov>, "
+                "Qi Wang <wqthu11@gmail.com>")
```

Comment:
Overall this is a pretty good organization of the featurizers, but we need to make an `external` subpackage like I mentioned for the composition featurizers.

Might also rename the current jarvis module to "composite" or something as the Jarvis featurizer is essentially a composite of several featurization techniques. I could see future featurizers being put into that module also.

Reply:
sure!